nncf/experimental/torch/nas/bootstrapNAS/training/stage_descriptor.py

```diff
@@ -51,6 +53,7 @@ def __init__(self, train_dims: List[ElasticityDim],
         self.bn_adapt = bn_adapt
         self.init_lr = init_lr
         self.epochs_lr = epochs_lr
+        self.sample_rate = sample_rate
```

Comment:
```suggestion
        if isinstance(sample_rate, int) and sample_rate <= 0:
            raise RuntimeError(f"Only positive integers are allowed for sample_rate, but sample_rate={sample_rate}.")
        self.sample_rate = sample_rate
```

We could prevent users from giving incorrect input.

Reply:
@vinnamkim Thanks for the suggestion! I have added a check for `sample_rate`.